charts/cryostat/values.yaml

```diff
@@ -198,6 +198,29 @@ authentication:
     ## @param authentication.basicAuth.filename Key within Secret containing the properties file. The properties file should contain one user per line, with the syntax "user=passHex", where "user" is the username and "passHex" is the SHA-256 hash of the desired password **(Required if basicAuth is enabled)**
     filename: ""
 
+oauth2Proxy:
+  ## @param authentication.oauth2Proxy.enabled Whether Cryostat should use OAuth2 Proxy for authentication. When true, Cryostat will authenticate users via OAuth2 Proxy.
+  enabled: true
+  ## @param authentication.oauth2Proxy.htpasswdFile Path to the htpasswd file for additional user authentication
+  htpasswdFile: "templates/auth_proxy_htpasswd"
+  ## @param authentication.oauth2Proxy.cookieSecret Secret key for encrypting OAuth2 Proxy cookies
+  cookieSecret: { { include "cryostat.defaultCookieSecret" . } }
+  image:
+    ## @param authentication.oauth2Proxy.image.repository Repository for the OAuth2 Proxy container image
+    repository: "quay.io/cryostat/oauth2-proxy"
```

Comment:
We don't host this image in our own repository namespace, it should be: `quay.io/oauth2-proxy/oauth2-proxy:latest`